### ISSUE TITLE
Highlight failing on certain rules

### ISSUE THREAD

**Issue by Arkkimaagi** [markalfred/SublimeLinter-contrib-stylint#5]
I'm having problems getting all the rules working. For example `zeroUnits` rule does not work.

It's missing the column in stylint for some reason:

```
81 zeroUnits warning Including the unit value is preferred
```

I thought that the SublimeLinter-contrib-stylint regex is not matching this properly:

``` Python
^(?P<line>\d+):(?P<near>\d+)\s*\w+\s*((?P<warning>warning)|(?P<error>error))\s*(?P<message>.+)$\s*
```

So I tried to alter it to

``` Python
^(?P<line>\d+)(:(?P<near>\d+))?\s*\w+\s*((?P<warning>warning)|(?P<error>error))\s*(?P<message>.+)$\s*
```

But I failed to test it properly, most likely because I do not quite understand how the plugin architecture works.

Any ideas on how to handle the errors without column?

**Comment by Arkkimaagi** [comment]
Now that I uninstalled the package control version of stylint and tried again with this fix, I got it working!

**Comment by markalfred** [comment]
This should be resolved via #6, thanks @Arkkimaagi, and let me know if you still have any issues 👍
